packages/lit-query/src/tests/type-inference.test.ts

```
@@ -0,0 +1,210 @@
+import {
+  dataTagSymbol,
+  QueryClient,
```

ESLint

[error] Member 'QueryClient' of the import declaration should be sorted alphabetically.

(sort-imports)